### PR TITLE
Add `Accomplishment` section

### PR DESCRIPTION
**Pull request by HenzelMoras**
### Description
im not sure but adding new section causes navbar issues

### Test Evidence
full width
navbar-size-prob

small window
appears-shrinked

### CODE REVIEW

**Comment by HenzelMoras** [comment]
@hossainemruz @cmarquezrusso whats the first check ?

**Comment by hossainemruz** [comment]
> @hossainemruz @cmarquezrusso whats Enforce Labels and Check Markdown links checks?
> how to enforce feature label?

`Enfore Labels` checks if any label has been assigned to this PR.  We generate release note automatically from the labels associate with the PRs. That's why we need to ensure that the PR has proper label. Otherwise, it won't be shown in the release note.

`Check Markdown  links` checks if there is any invalid link in the markdown files used in this repository.

**Comment by hossainemruz** [comment]
>im not sure but adding new section causes navbar issues

Don't show it in the navbar. Set `hideTitle: true` in the data file. Here, is an example: https://github.com/hugo-toha/toha/blob/1c3154d47a3433d0736c3e8d49cc54f3de017b3a/exampleSite/data/en/sections/recent-posts.yaml#L9


Also, please add the example data file here: https://github.com/hugo-toha/toha/tree/main/exampleSite/data/en/sections

**Comment by HenzelMoras** [comment]
>> Don't show it in the navbar. Set `hideTitle: true` in the data file. Here, is an example:
> i setted showNavbar: false and it seems to be working

>> Also, please add the example data file here: https://github.com/hugo-toha/toha/tree/main/exampleSite/data/en/sections
> i push the data file recently

**Comment by hossainemruz** [comment]
Let me know when this PR is ready to review. Update the title of this PR to "Add `Accomplishment` section"

**Comment by HenzelMoras** [comment]
> >Let me know when this PR is ready to review. Update the title of this PR to "Add `Accomplishment` section"
> okay
im finding it hard to slice sections[6: ] to dropdown in go templating language

**Comment by hossainemruz** [comment]
>im finding it hard to slice sections[6: ] to dropdown in go templating language
in nabar.html

Why it is necessary?

**Comment by HenzelMoras** [comment]
> > im finding it hard to slice sections[6: ] to dropdown in go templating language
> > in nabar.html
> 
> > Why it is necessary?
well if we want to display elements on navbar

**Comment by hossainemruz** [comment]
You can probably use this function: https://gohugo.io/functions/last/

Probably something like this.

```
last (sub (len COLLECTION) 6)  COLLECTION
```

**Comment by HenzelMoras** [comment]
@hossainemruz 

achievments-dropdown
Recent-posts

**Comment by HenzelMoras** [comment]
> @hossainemruz
> 
> achievments-dropdown
> Recent-posts



> @hossainemruz
> 
> achievments-dropdown
> Recent-posts

**Comment by HenzelMoras** [comment]
@hossainemruz  see if u can make changes

**Comment by hossainemruz** [comment]
@HenzelMoras I am making some changes in the PR.

**Comment by HenzelMoras** [comment]
>  @HenzelMoras I am making some changes in the PR.

@hossainemruz  i made some too  kindly check
i hope it helps

**Comment by HenzelMoras** [comment]
@hossainemruz  i see certain peices of code repeated multiple times 
```
{{ $sectionID := replace (lower .section.name) " " "-"  }}
{{ if .section.id }}
  {{ $sectionID = .section.id }}
{{ end }}
```

**Comment by HenzelMoras** [comment]
> @hossainemruz i see certain peices of code repeated multiple times
> 
> ```
> {{ $sectionID := replace (lower .section.name) " " "-"  }}
> {{ if .section.id }}
>   {{ $sectionID = .section.id }}
> {{ end }}
> ```

maybe we could create shortcodes

File: layouts/partials/navigators/navbar.html [hunk 1]
```diff
@@ -56,16 +56,39 @@
...
+                    {{ $sectionID = .section.id }}
+                  {{ end }}
+                  {{ if lt .section.weight 7 }}
+                    <li class="nav-item">
+                      <a class="nav-link" href="#{{ $sectionID }}">{{ .section.name }}</a>
+                    </li>
+                  {{ end }}
             {{ end }}
           {{- end }}
+          {{ if gt (len $sections) 6 }}
```
Comment (HenzelMoras) [inline layouts/partials/navigators/navbar.html:71]: View this part

File: layouts/partials/navigators/navbar.html [hunk 2]
```diff
@@ -56,16 +56,39 @@
...
+                      {{ $sectionID = .section.id }}
+                    {{ end }}
+                    {{ if gt .section.weight 6 }}
+                      <a class="dropdown-item" href="#{{ $sectionID }}">{{ .section.name }}</a>
+                    {{ end }}
+                  {{ end }}
+                {{- end }}
+              </div>
+            </li>
+
```
Comment (HenzelMoras) [inline layouts/partials/navigators/navbar.html:91]: Till here

**Comment by hossainemruz** [comment]
@HenzelMoras I have pushed some changes. Can you check?

**Comment by hossainemruz** [comment]
This is how your data file should look like.
https://github.com/hugo-toha/hugo-toha.github.io/pull/50/files

**Comment by HenzelMoras** [comment]
> @HenzelMoras I have pushed some changes. Can you check?

@hossainemruz  on it

**Comment by HenzelMoras** [comment]
> > @HenzelMoras I have pushed some changes. Can you check?
> 
> @hossainemruz on it

how can i check this locally :sweat_smile: ?

**Comment by hossainemruz** [comment]
@HenzelMoras  I have added a helper function for generating section id.

**Comment by HenzelMoras** [comment]
> @HenzelMoras I have added a helper function for generating section id.

how can i check this locally sweat_smile ?

**Review by HenzelMoras — COMMENTED** [review]
@hossainemruz great much better

**Review by hossainemruz — APPROVED** [review]
Thank you @HenzelMoras . Looking good so far. Merging....

**Comment by hossainemruz** [comment]
Merged.

**Comment by HenzelMoras** [comment]
> Merged.

@hossainemruz  noice

**Comment by hossainemruz** [comment]
I owe you an apology for this PR. I merged this PR without squashing. As  a result all the commits of this PR get merged into main. When I tried to pull the changes in my another PR, it was causing conflict at every commit. In order to fix that, I had to reset main branch. After the reset, your contribution in this PR was showing in my name. I wasn't able to restore you as a contributor of this PR. Now, your contribution in this theme showing 2 commits where it should be 3. I am really sorry about this.

**Comment by HenzelMoras** [comment]
> I owe you an apology for this PR. I merged this PR without squashing. As a result all the commits of this PR get merged into main. When I tried to pull the changes in my another PR, it was causing conflict at every commit. In order to fix that, I had to reset main branch. After the reset, your contribution in this PR was showing in my name. I wasn't able to restore you as a contributor of this PR. Now, your contribution in this theme showing 2 commits where it should be 3. I am really sorry about this.

@hossainemruz  no problem happens with everyone

**Pull request merged**